internal/cluster/cluster.go

```diff
@@ -1532,3 +1533,59 @@ func (m *Manager) updateSchedulableMastersForcedTrue(ctx context.Context, cluste
 
 	return nil
 }
+
+func (m *Manager) HandleVerifyVipsResponse(ctx context.Context, clusterID strfmt.UUID, stepReply string) error {
+	log := logutil.FromContext(ctx, m.log)
+	return m.db.Transaction(func(tx *gorm.DB) error {
+		var response models.VerifyVipsResponse
+		if err := json.Unmarshal([]byte(stepReply), &response); err != nil {
+			log.WithError(err).Error("HandleVerifyVipsResponse: unmarshal")
+		}
+		cluster, err := common.GetClusterFromDB(common.LoadTableFromDB(common.LoadTableFromDB(tx, common.APIVIPsTable), common.IngressVIPsTable),
+			clusterID, common.SkipEagerLoading)
+		if err != nil {
+			log.WithError(err).Errorf("HandleVerifyVipsResponse: getting cluster %s", clusterID.String())
+			return err
+		}
+		updated := false
+		for _, v := range response {
+			vipResponse := v
```

Comment:
why do we need a new var here? can't we just rename the original var?

Reply:
Linter produces an error for loop variables that are passed to functions.

Reply:
> Linter produces an error for loop variables that are passed to functions.

Oh right. So I guess gofunk.ForEach worth a try also here.

Reply:
Seems quite a big function to put in ForEach.  Not sure it makes it better.